pyproject.toml

```diff
@@ -31,11 +31,13 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```

Comment:
I'm slightly hesitant about the potential impact of this change - we *shouldn't* have any users running Python 3.6.0 or 3.6.1 in this day and age, but are we confident enough in that assertion to make this change?

Reply:
I did this based on the black requirement, happy to pull this in from your other PR #1069